pkg/volume/rbd/rbd_util.go

```diff
@@ -23,6 +23,7 @@ package rbd
 
 import (
 	"encoding/json"
+	serrors "errors"
```

Comment:
serrors?

Reply:
@liggitt we have an another import 	"k8s.io/apimachinery/pkg/util/errors" here, so thought of using std errors package directly.

Reply:
goerrors might be clearer, not a big deal

Reply:
sure. let me reword to `goerrors`